### ISSUE TITLE
Disable myget package deployments for PRs

### ISSUE THREAD

**Issue by pouladpld** [TelegramBots/telegram.bot#459]
Since anyone can make a PR from a fork, it is possible for them to include malicious code it and make a PR to this repo. MyGet shouldn't deploy packages on PR builds

**Comment by pouladpld** [comment]
For the similar reasons, Travis-CI is configured not to run tests on PRs.

**Comment by Olfi01** [comment]
But isn't the point of myget having the compiled code of any state readily available? It's up to the users to check whether there's malicious code in what they are using

**Comment by pouladpld** [comment]
No! It's up to us to ensure security and accuracy of the product. If we do not so, we loose reliability and trust of the users.
However, once a PR is approved and merged into develop, the deployment occurs automatically.

**Comment by Olfi01** [comment]
I'm too tired to break up the whole security hypersensibility discussion yet again... So go on, do whatever you think is best

**Comment by pouladpld** [comment]
Fixed in <https://github.com/TelegramBots/telegram.bot/commit/85e0cc146494f6318e559781f831719f1bfad0fc>
